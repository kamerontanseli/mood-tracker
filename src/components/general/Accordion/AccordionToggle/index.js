import React from "react";
import Button from '../../Button'

const AccordionToggle = ({ toggled, ...props }) => (
  <Button {...props}>
    {toggled ? <span>&uarr;</span> : <span>&darr;</span>}
  </Button>
);

export default AccordionToggle;
