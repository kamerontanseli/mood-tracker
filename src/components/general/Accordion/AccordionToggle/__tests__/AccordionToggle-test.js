import React from 'react'
import renderer from 'react-test-renderer'
import AccordionToggle from '..'

describe('render', () => {
  it('should render', () => {
    const snapshot = renderer.create(
      <AccordionToggle />
    );
    expect(snapshot).toMatchSnapshot()
  })
  it('should render when toggled', () => {
    const snapshot = renderer.create(
      <AccordionToggle toggled />
    );
    expect(snapshot).toMatchSnapshot()
  })
})