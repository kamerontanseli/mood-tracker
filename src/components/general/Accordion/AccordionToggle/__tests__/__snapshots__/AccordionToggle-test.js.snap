// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render should render 1`] = `
<button
  className="Button"
>
  <span>
    ↓
  </span>
</button>
`;

exports[`render should render when toggled 1`] = `
<button
  className="Button"
>
  <span>
    ↑
  </span>
</button>
`;
